### PR TITLE
Default to dark theme when prefers-color-scheme set to dark

### PR DESCRIPTION
**Pull request by patcito**
default to dark theme if window?.matchMedia('(prefers-color-scheme: dark)')

### CODE REVIEW

**Review by turtlemoji — DISMISSED** [review]

File: src/client/themes/index.ts [hunk 1]
```diff
@@ -21,6 +21,8 @@ export const getTheme = (theme?: Theme): DefaultTheme => {
     case 'explorer':
       return explorerTheme;
     default:
+      const isDarkMode: MediaQueryList = window?.matchMedia('(prefers-color-scheme: dark)');
+      if (isDarkMode.matches) return darkTheme;
       return lightTheme;
```
Comment (karelianpie) [inline src/client/themes/index.ts:26]: nit
```suggestion
      return isDarkMode.matches ? darkTheme : lightTheme;
```

**Review by karelianpie — APPROVED** [review]
LGTM, minor nit for readability

File: src/client/themes/index.ts (hunk 1 above)
Comment (karelianpie) [reply src/client/themes/index.ts:26]: or
```suggestion
      const { matches: prefersColorSchemeDark } = window?.matchMedia('(prefers-color-scheme: dark)');
      return prefersColorSchemeDark ? darkTheme : lightTheme;
```
so the variables names are more in line with what the code is doing

**Review by turtlemoji — DISMISSED** [review]

File: src/client/themes/index.ts [hunk 2]
```diff
@@ -21,6 +21,8 @@ export const getTheme = (theme?: Theme): DefaultTheme => {
     case 'explorer':
       return explorerTheme;
     default:
-      return lightTheme;
+      const isDarkMode: MediaQueryList = window?.matchMedia('(prefers-color-scheme: dark)');
```
Comment (karelianpie) [inline src/client/themes/index.ts:24]: No longer needed

```suggestion
```

**Review by karelianpie — APPROVED** [review]

**Review by turtlemoji — APPROVED** [review]

**Pull request merged**